### ISSUE TITLE
v3.7.1 Unexpected token

### ISSUE THREAD

**Issue by zie9024** [sampotts/plyr#2472]
I import 3.7.1 I get the following error:

```
Module parse failed: Unexpected token (1:25052)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file.
```

**Comment by Matthew10r** [comment]
same here..

**Comment by sampotts** [comment]
Found the issue. I need to add `@babel/plugin-proposal-optional-chaining` to the build. I'll get 3.7.2 out shortly.

**Comment by sampotts** [comment]
Should be fixed in v3.7.2
